### ISSUE TITLE
Add Royaltea

### ISSUE THREAD

**Issue by arch0345** [osmlab/name-suggestion-index#6714]
~~[According to their US website](https://www.royalteaus.com/about), they have over 130 locations in China~~. They have a few locations in the US and [looking at their website for Vietnam](http://www.royaltea.vn/en/royaltea-1/), also have some locations in Southeast Asia.

**Comment by LaoshuBaby** [comment]
I must say I was catched by "China" in first line. 😀

**TL:DR: They stop business in China**

<details>
<summary>My research on Chinese Internet</summary>
According to my research on Chinese Internet, the "Royaltea 皇茶" seems stop their business in China, Their zh-Hans Website stop update in 2018-08, and their Weibo (A China Twitter) stop update in 2019, their Wechat Official Account stop management in 2020, and on takeaway platform in China such as 大众点评 I can only found some other "皇茶", but none of them named after "Royaltea".

Looks like the latest Royaltea shop in HK also closed: https://www.bastillepost.com/hongkong/article/10576390-%E6%97%BA%E8%A7%92%E8%8C%B6%E9%A3%B2%E5%BA%97%E5%AE%A3%E5%B8%83%E7%B5%90%E6%A5%AD-%E8%B2%A0%E8%B2%AC%E4%BA%BA%E6%9B%BE%E8%A2%AB%E5%9C%8B%E5%AE%89%E8%99%95%E5%B8%B6%E8%B5%B0%E8%AA%BF%E6%9F%A5

I visited their royaltea.hk FB/INS account and looks like they were tagged as “HK independence” or "Yellow Corpse" so I don't think they can still run any business in China. 

<b>This paragraph explain why I don't think they have shop in CN/HK/MO. Some commercial map service such as Baidu still have POI about "Royaltea 皇茶", but the data and streetview is in 2020, too old.</b>
</details>

<s>For Vietnam's website may need Minh's language help 😂</s>
(But I guess it work, I found `name=Cà Phê Royaltea` and `name:vi=Royaltea` in Vietnam)

**Comment by arch0345** [comment]
Thanks for your research @LaoshuBaby, that explains why I couldn't find a website for China/Hong Kong
